shiny/reactive/__init__.py

```diff
@@ -26,8 +29,11 @@
     "on_flushed",
     "poll",
     "file_reader",
+    "value",
     "Value",
+    "calc",
     "Calc",
+    "effect",
     "Effect",
```

Comment:
Should the old names be removed and pyright ignored above?

```suggestion
    "calc",
    "effect",
```

Reply:
I think we need to keep the old names for backward compatibility, at least for the time being. If we were to remove them now, then a lot of user code would get flagged by pyright.